Broken demo Virtualbox

The `v20161218.1` version of the Virtualbox Vagrant demo never gets past the SSH portion of `vagrant up`. I think something is wrong with the ssh keys or ssh configuration.

I am still seeing the same issue:

```
make init
CLUSTER_DOMAIN="demo.localflynn.com" SSH_AUTH_SOCK="" FLAG_BOOTSTRAP=false GITLAB_FLAG="true" GITLAB_SWAP="0" GITLAB_HOSTNAME="demo.localflynn.com" GITLAB_PORT="8060" vagrant box update
==> default: Checking for updates to 'flynn-base'
    default: Latest installed version: 20161218.1
    default: Version constraints: > 0
    default: Provider: virtualbox
==> default: Updating 'flynn-base' with provider 'virtualbox' from version
==> default: '20161218.1' to '20161229.1'...
==> default: Loading metadata for box 'https://dl.flynn.io/vagrant/flynn-base.json'
==> default: Adding box 'flynn-base' (v20161229.1) for provider: virtualbox
    default: Downloading: https://dl.flynn.io/vagrant/boxes/flynn_v20161229.1_ubuntu-xenial_virtualbox.box
    default: Calculating and comparing box checksum...
==> default: Successfully added box 'flynn-base' (v20161229.1) for 'virtualbox'!
make[1]: Entering directory `/home/users/bret/Projects/satorix-integration-test'
CLUSTER_DOMAIN="demo.localflynn.com" SSH_AUTH_SOCK="" FLAG_BOOTSTRAP=false GITLAB_FLAG="true" GITLAB_SWAP="0" GITLAB_HOSTNAME="demo.localflynn.com" GITLAB_PORT="8060" vagrant up --provider virtualbox
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'flynn-base'...
==> default: Importing base box 'flynn-base'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'flynn-base' is up to date...
==> default: Setting the name of the VM: satorix-integration-test_default_1483147488866_46093
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.
make[1]: *** [up] Error 1
make[1]: Leaving directory `/home/users/bret/Projects/satorix-integration-test'
make: *** [init] Error 2
```

--- a/Vagrantfile
+++ b/Vagrantfile
@@ -49,6 +49,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
     # RFC 5737 TEST-NET-1 used to avoid DNS rebind protection
     override.vm.network "private_network", ip: "192.0.2.100"
+
+    # Workaround for https://www.virtualbox.org/ticket/15705
+    v.customize ["modifyvm", :id, "--cableconnected1", "on"]
   end
 
   config.vm.provider "vmware_fusion" do |v, override|

--- a/demo/Vagrantfile
+++ b/demo/Vagrantfile
@@ -35,6 +35,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provider "virtualbox" do |v|
     v.memory = ENV["VAGRANT_MEMORY"] || 2048
     v.cpus = ENV["VAGRANT_CPUS"] || 2
+
+    # Workaround for https://www.virtualbox.org/ticket/15705
+    v.customize ["modifyvm", :id, "--cableconnected1", "on"]
   end
 
   config.vm.provider "vmware_fusion" do |v|

